docs/docs/api/arg.md

```diff
@@ -4,11 +4,24 @@ title: Arg
 sidebar_label: Arg
 ---
 
```

Comment:
Can you add just like on the Producer page the following line at the beginning of the page (for every API keyword):

```
import { Arg } from "@c11/engine.macro"
```

Otherwise, it might be confusing where this keyword springs up from